enterprise/coderd/workspaces_test.go

```diff
@@ -641,9 +642,13 @@ func TestWorkspacesFiltering(t *testing.T) {
 
 		dormantTTL := 24 * time.Hour
 
+		db, ps := dbtestutil.NewDB(t, dbtestutil.WithTimezone("UTC"))
```

Comment:
I see that you enabled UTC in a few tests. Should it be enabled by default?

Reply:
I'm setting the default to `Canada/Newfoundland` instead. This preserves determinism but introduces enough weirdness to surface timezone bugs.

Reply:
If we want to regularly catch tz issues, then I think something other than UTC is preferable. All our deployments and default configurations use UTC, so this is the only way I imagine we'd catch it.

Reply:
Ok, thanks for the justification. I'm only concerned about developers blindly copying `dbtestutil.NewDB(t, dbtestutil.WithTimezone("UTC"))` everytime they need to write a test, and actually apply a workaround for using a non-standard TZ.

Reply:
Added a linter rule.